docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Use a custom domain to expose a service
+---
+
+This tutorial shows how to set up your custom domain and prepare a certificate to use the domain for exposing a service. The components used are Gardener [External DNS Management](https://gardener.cloud/docs/concepts/networking/dns-managment/#external-dns-management) and [Certificate Management](https://gardener.cloud/docs/concepts/networking/cert-managment/).
+
+To learn how to expose a service, go to [this](./apix-01-expose-service-apigateway.md) tutorial.
+
+## Prerequisites
+
+If you use a cluster not managed by Gardener, install the required components manually. Follow these steps:
+
+1. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+2. Download the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) projects locally.
+
+3. Enable vertical Pod autoscaling on your cluster. For example, for Google Cloud Platform, run:
+
+   ```bash
+   gcloud beta container clusters update {PROJECT_NAME} --enable-vertical-pod-autoscaling
```

Comment:
This doesn't work at all - there should be CLUSTER_NAME instead of PROJECT_NAME.
This is the command that works on my machine:
`gcloud container clusters update {CLUSTER_NAME} --enable-vertical-pod-autoscaling --zone="{CLUSTER_ZONE}" --project="{PROJECT_NAME}"`

Reply:
Project and Zone can be pre-configured so that gcloud knows it and user doesn't have to provide them explicitly, but the steps for setting them are not in the tutorial.

Reply:
@Tomasz-Smelcerz-SAP You are right, it was probably my own workspace configuration that allowed me shortening the command. Maybe it would be better to refer to their docs (https://cloud.google.com/sdk/gcloud/reference/beta/container/clusters/update) but still it depends on which cluster the component is about to run.

Reply:
@mmitoraj gcloud tool configuration. But it's probably better to have it explicitly in the command, as the one I pasted.